src/utilstrencodings.cpp

```diff
@@ -506,3 +506,25 @@ bool ParseFixedPoint(const std::string &val, int decimals, int64_t *amount_out)
     return true;
 }
 
+/// Parse the decimal string into a little-endian byte vector.
+std::optional<std::vector<unsigned char>> ParseArbitraryInt(const std::string& num_string)
+{
+    std::vector<unsigned char> result;
+    size_t start = num_string.find_first_not_of(' ');
```

Comment:
Should this perhaps use a more robust mechanism for stripping whitespace, that also accounts for tabs, etc?